build_scripts/es_setup/install_configuration.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#install_demo_configuration.sh [-y]
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo "## Search Guard Demo Installer ##"
+echo "Warning: Do not use on production or public reachable systems"
```

Comment:
Can we write one that makes sense for our production environment?

Reply:
I packaged this script inside the search-guard-5.zip to streamline and make it cleaner. Also updated the script to better reflect an OpenRecords production environment.